server/src/serializers/AttractionSerializer.js

```diff
@@ -0,0 +1,25 @@
+import ReviewSerializer from "./ReviewSerializer.js";
+class AttractionSerializer {
+  static async getSummary(attraction) {
+    const allowedAttributes = ["id", "name", "description"];
+    let serializedAttraction = {};
+    for (const attribute of allowedAttributes) {
+      serializedAttraction[attribute] = attraction[attribute];
+    }
+    return serializedAttraction;
+  }
+  static async getDetails(attraction) {
```

Comment:
It's great for readability to add a new empty line between functions! (so after line 10, before getDetails is defined)